src/components/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div>
+    <confirm-dialog
+      ref="confirmTranslationDialog"
+    />
+    <!-- Name Translation form -->
+    <v-form v-model="nameTranslationForm" class="name-translation-form">
+      <v-row>
+        <v-col class="pb-0">
+          <v-text-field
+            filled
+            persistent-hint
+            label="Name Translation"
+            id="name-translation-input"
+            v-model="nameTranslation"
+            :rules="nameTranslationRules">
+          </v-text-field>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col>
+          <div class="action-btns">
+          <v-btn large color="primary"
+            id="name-translation-btn-ok"
+            :disabled="!nameTranslationForm"
+            @click="addTranslation"
+          >
+            OK
```

Comment:
FYI, this renders (in the DOM) "OK" on a separate line, which makes it trickier to unit test because you can't select just the text so you can't use `toBe()` and instead have to use the less precise `contains()` to verify it.

This may not matter here, but that's why we either have the button text on the same line as the button element, or within its own span (which we can select during testing), allowing us to verify the text exactly.